docs/developer/extending/apps.mdx

```diff
@@ -48,29 +48,32 @@ If there are no errors in the response, a new app is created:
 ```
 
 The response contains the `authToken` field - this is the token that the app needs to include in requests to Saleor API to authorize itself and have access to queries or mutations that it needs. Apps can authorize in Saleor using [the Authorization header, just as regular users](developer/users.mdx#authentication).
+:::note
```

Comment:
I would replace this note with a full header example like the one we have for JWT